calico-cloud/reference/resources/felixconfig.mdx

```diff
@@ -171,7 +171,7 @@ spec:
-| bpfDataIfacePattern                   | In eBPF dataplane mode, controls which interfaces Felix should attach BPF programs to in order to catch traffic to/from the external network. This needs to match the interfaces that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to NodePorts and services from outside the cluster. It should not match the workload interfaces (usually named cali...).. This is a tech preview feature and subject to change in future releases.                                                                                                                                                                                                                                                                        | regular expression                                                 | string                                  | `^(en.\*                                                                                                                                                                                                                                                                                                                                                                                                                                          | eth.\* | tunl0$)` |
+| bpfDataIfacePattern                   | In eBPF dataplane mode, controls which interfaces Felix should attach BPF programs to in order to catch traffic to/from the external network. This needs to match the interfaces that Calico workload traffic flows over as well as any interfaces that handle incoming traffic to NodePorts and services from outside the cluster. It should not match the workload interfaces (usually named cali...).. This is a tech preview feature and subject to change in future releases.                                                                                                                                                                                                                                                                        | regular expression                                                 | string                                  | ^(en&#46;&#42;&#124;eth&#46;&#42;&#124;tunl0$)                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
The only thing missing is having the whole thing in code font. But whenever I try that, it keeps the codes as-is.

Let's go ahead and merge this. I've added [a bug ticket](https://tigera.atlassian.net/browse/DOCS-1273) for the rest.